rasa_core/server.py

```diff
@@ -534,6 +551,38 @@ def create_app(agent=None,
                                 "Evaluation could not be created. Error: {}"
                                 "".format(e))
 
+    @app.post("/intentEvaluation")
+    @requires_auth(app, auth_token)
+    async def evaluate_intents(request: Request):
+        """Evaluate intents against a Rasa NLU model."""
+
+        tmpdir = tempfile.mkdtemp()
+
+        zipped_model_path = os.path.join(tmpdir, 'model.tar.gz')
```

Comment:
this sounds like something we are also doing in the training endpoint, probably makes sense to move the "take body and write to file" into a function

Reply:
hmm, we don't dump the whole body in the training endpoint, but just the json payload. will create a function though